connectors/src/connectors/google_drive/index.ts

```diff
@@ -769,15 +770,15 @@ export async function setGoogleDriveConfig(
       new Error(`Google Drive config not found with connectorId ${connectorId}`)
     );
   }
+
+  if (!["true", "false"].includes(configValue)) {
+    return new Err(
```

Comment:
I would check the type of configValue in each switch case, since it could be anything, but it works for now.